src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -320,6 +324,9 @@ private static String getFullyQualifiedClassName(String filePath, String moduleN
                 "com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck");
         moduleMappings.put("LineLength",
                 "com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck");
+        moduleMappings.put("Checker",
+                "com.puppycrawl.tools.checkstyle.CheckerCheck");
+
```

Comment:
Had to add Checker here, cuz some tests started throwing an error something like: 

`config comment is not defined properly `